docs/rfcs/0005-json-library.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: RFC process
+permalink: /rfc/0005-json-library
+---
+
+* Start date: 2020-07-28
+* RFC PR: TBD
+
+## Summary
+
+Introduce a library for reading and writing JSON into Android Components and Fenix.
+
+## Motivation
+
+We currently interact with JSON in many areas of Android Components and Fenix. Currently serialization and deserialization is all done manually with `JSONObject`, which creates more work for developers and potential bugs. Fenix also is interested in using a library and we'd like to avoid shipping multiple JSON libraries to users. Using a JSON library allows us to reduce code that needs to be written, potentially gain some performance wins from a well maintained lib, and avoid bugs in our custom JSON implementations.
```

Comment:
> Fenix also is interested in using a library and we'd like to avoid shipping multiple JSON libraries to users.

Note that this is something we would want to allow for all consuming apps. Using no library in AC also allows Fenix to make a choice and not ship multiple libraries. :)